addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -3634,6 +3754,44 @@ class SnippetsMenu extends Component {
         });
         this.draggableComponent = useDragAndDrop({ ref: { el: this.el }, ...dragAndDropOptions });
     }
+    /**
+     * Gets the selectors that determine where the snippet can be placed.
+     *
+     * @private
+     * @param {jQuery} $baseBody
+     * @return {Object} selectors
+     */
+    _getSelectors($baseBody) {
+        let $selectorSiblings = $();
+        let $selectorChildren = $();
+        const selectorExcludeAncestor = [];
+        var temp = this.templateOptions;
+        for (var k in temp) {
```

Comment:
Seeing it's in the diff anyway, you can take the opportunity to change `var`s with `let`/`const`.